src/toil_scripts/batch_alignment/bwa_alignment.py

```diff
@@ -258,6 +256,23 @@ def tarball_files(work_dir, tar_name, uuid=None, files=None):
                 f_out.add(os.path.join(work_dir, fname), arcname=fname)
 
 
+def download_from_s3_url(file_path, url):
```

Comment:
We determined that the "common library" PR will not be included in the 1.0 release.